pulp_container/app/serializers.py

```diff
@@ -505,6 +508,38 @@ class ManifestCopySerializer(CopySerializer):
     )
 
 
+class RemoveImageSerializer(serializers.Serializer):
+    """
+    A serializer for parsing and validating data associated with the image removal.
+    """
+
+    digest = serializers.CharField(help_text="sha256 of the Manifest file")
+
+    def validate(self, data):
+        """
+        Validate and extract the latest repository version, manifest, and tags from the passed data.
+        """
+        new_data = {}
+        new_data.update(self.initial_data)
+
+        latest_version = new_data["repository"].latest_version()
```

Comment:
if you subclass it from TagOperationSerializer you would not have to have this check.

Reply:
not really, TagOperationSerializer has tag field, disregard this comment.